pkg/endpoint/endpoint.go

```diff
@@ -27,13 +27,13 @@ import (
 )
 
 const (
-	KineSocket       = "unix://kine.sock"
-	SQLiteBackend    = "sqlite"
-	DQLiteBackend    = "dqlite"
-	ETCDBackend      = "etcd3"
-	JetStreamBackend = "jetstream"
```

Comment:
Don't drop this; need to keep it for compat

Reply:
👍 Good catch, I read the [comments in driver](https://github.com/k3s-io/kine/pull/158/files#diff-98f825ac9c2de7a75a0317fae315becccbf9c7dbff1a2d87444e15915dde4f81L58-L60) as it being `nats://` but you are right, it derives from the URL. Added back.